package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "embeddedchat",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
can you undo this package-lock changes and I think this will be good to go.

Reply:
Done! Please have a look.